src/relay/chit_brickFamiliar.ash

```diff
@@ -2109,6 +2109,12 @@ void bakeFamiliar() {
 		boolean candyBagDropped = get_property("_bagOfCandy").to_boolean();
 		info += candyBagDropped ? "bag dropped today" : "can drop bag today";
 		break;
+	case $familiar[grey goose]:
+		int famweight = familiar_weight(myfam);
+		info += famweight + " lb";
+		int target = max(famweight+1, 6);
+		if(famweight < 20)
+			info += " - " +  (target**2 - myfam.experience) + " exp to " + target + " lb";
```

Comment:
This seems of questionable value to me since all this information is already available, but at the same time it also seems harmless so I guess I'm cool with it. Not to mention it makes it available without having to mouse over anything, so I guess I can see how it would be useful after all.

Reply:
Yeah, I couldn't figure out a way to get it to put the familiar and equipment on opposite sides when there is enough room, or I would have done that to make it more backwards compatible. But honestly, I actually like having the familiar and equipment side-by-side better. What do you like better about the previous view? Do you think that providing an updated default layout that doesn't have a full-width familiar brick would be a fix worth considering?

I don't have a Mumming Trunk, so I didn't even know that it shows an icon on your familiar brick. It's lucky that the changes were an unexpected improvement. It might be something in the CSS rather than in the HTML.

The Grey Goose showing unbuffed weight and exp to next lb was a big QoL thing when I was running Grey You runs. I think it will be valuable information-at-a-glance to many users.